src/main/resources/cli-help/ballerina-dist-clean.help

```diff
@@ -0,0 +1,9 @@
+NAME
+       ballerina-dist-clean - Clean available distributions except active distribution.
+
+SYNOPSIS
+       ballerina dist clean
+
+DESCRIPTION
+       Clean removes the ballerina distributions from your local environment except the active distribution.
```

Comment:
@suganyasuven 
We need to improve the description. Isn't there better way to specify instead of "Clean removes"
@praneesha @MaryamZi

Reply:
It would be sufficient to say "Removes the Ballerina distributions from your local environment except the active distribution."

Reply:
Sorry, just saw this. The convention with command help text description is to start with the command name. 

Few examples:
```cmd
$ ballerina run -h
DESCRIPTION
       Run builds and executes the given .bal file, module, or program.

$ ballerina pull -h
DESCRIPTION
       Pull downloads the specified module from Ballerina Central
       along with its dependencies. It then caches this module at
       '.ballerina' directory in user home.

$ ballerina build -h
DESCRIPTION
       Build compiles Ballerina sources and produces executable JAR file(s). However, 
       if the module does not contain an entry point, it will produce a non-executable
       JAR file, which can be used in another module/program.
```

Shall we fix https://github.com/ballerina-platform/ballerina-update-tool/pull/2/files#diff-015d1677bdbaaee1a9b08c9eb231c522R9?

Reply:
Made the suggested changes with this [PR](https://github.com/ballerina-platform/ballerina-update-tool/pull/8)

Reply:
It doesn't make sense to say "Remove deletes". I don't think there is a standard convention to have the description in this way.

It is ok to do this change just for the purpose of consistency. However, IMO, we should change this for all commands to one of the following:

The "remove" command deletes.
Just say "Deletes...." as it is obvious that we are referring to the "remove" command.
@suganyasuven - I will create an issue to correct this for all commands.